docs/_source/docs/hooks.rst

```diff
@@ -152,26 +152,28 @@ custom_hook.py
         argument: str
             The argument is available from the base class and contains the
             argument defined in the Sceptre config file (see below)
-        stack: sceptre.stack.Stack
-             The associated stack of the hook.
-        connection_manager: sceptre.connection_manager.ConnectionManager
-            Boto3 Connection Manager - can be used to call boto3 api.
 
         """
         def __init__(self, *args, **kwargs):
             super(CustomHook, self).__init__(*args, **kwargs)
 
-        def run(self):
+        def run(self, stack):
```

Comment:
thanks for the contribution however I don't think we can break every Sceptre custom hook out in the wild.  I would prefer a more general solution that would allow hooks or resolvers to access the stack object.  

If this is the only solution then i think it would need to be done with a new interface and then deprecating the old one.